cpp/pipelines/add_superior_and_alertable_flags.cc

```diff
@@ -101,11 +102,24 @@ void ProcessRecord(MARC::Writer * const marc_writer, const std::unordered_set<st
 
 
 void AddSuperiorFlag(MARC::Reader * const marc_reader, MARC::Writer * const marc_writer,
-                     const std::unordered_set<std::string> &superior_ppns)
+                     const std::unordered_set<std::string> &superior_ppns, const std::string &flavour)
 {
     unsigned modified_count(0);
+    
+    Zeder::SimpleZeder zeder(Zeder::IXTHEO, { "eppn", "pppn", "kat" });
```

Comment:
As discussed => please extend SimpleZeder to allow an optional arg as well that will be used as filter_regexps, and use it to compare kat=IxTheo.